app/workers/syncsIn.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+require_once __DIR__ . '/../../vendor/autoload.php';
+
+use Utopia\App;
+use Utopia\Cache\Adapter\Redis as RedisCache;
+use Utopia\Queue;
+use Utopia\Queue\Message;
+
+require_once __DIR__ . '/../init.php';
+
+define("CURRENT_REGION", App::getEnv('_APP_REGION', 'nyc1'));
```

Comment:
Defining constants in multiple files in the code is not suggested. We can just use the env var directly. Assuming `nyc` is the default region is wrong. We can have `default` as the default region.